benchmarks/functional_autograd_benchmark/functional_autograd_benchmark.py

```diff
@@ -6,6 +6,13 @@
 from collections import defaultdict
 from typing import NamedTuple, Callable, List, Any
 
+try:
+    import functorch as ft
+    has_functorch = True
+    print(f"Found functorch: {ft.__version__}")
```

Comment:
spurious print?

Reply:
I was using this for checking with different functorch versions and output logs. If you think this should be removed, I have no strong opinion on keeping this.